### ISSUE TITLE
Excluding a feature present on only some crates in a workspace displays warning

### ISSUE THREAD

**Issue by leighmcculloch** [taiki-e/cargo-hack#157]
Excluding a feature that is not on all crates in a workspace displays warning for each crate that doesn't have the feature.

For example, in the following repo the feature `docs` is defined on the `stellar-contract-sdk` crate, but no others. When excluding the feature a warning displays for all other repos:
```
$ git clone https://github.com/stellar/rs-stellar-contract-sdk
$ cd rs-stellar-contract-sdk
$ git checkout 9481010ac820ce3694d5853db79beee38e302690
$ cargo hack --feature-powerset --exclude-features docs test
warning: specified feature `docs` not found in package `example_add_i32`
warning: specified feature `docs` not found in package `example_add_i64`
warning: specified feature `docs` not found in package `example_udt`
warning: specified feature `docs` not found in package `example_contract_data`
warning: specified feature `docs` not found in package `example_create_contract`
warning: specified feature `docs` not found in package `example_linear_memory`
...
```

**Comment by taiki-e** [comment]
Thanks! Addressed in 0.5.16.

**Comment by leighmcculloch** [comment]
Thanks for fixing! ❤️
